app/views/users/show.html.erb

```diff
@@ -17,4 +17,4 @@
   <% end %>
 </div>
 
-<%= link_to "See all posts", "/users/#{@user.id}/posts?page=1" %>
\ No newline at end of file
+<%= link_to "See all posts", user_posts_path(@user, page: 1) %>
```

Comment:
- [ ] You have actually been at your best but the `see all post` router is still not working yet as indicated by previous reviews. I suspect that the pagination is the problem

![Screenshot from 2023-11-04 13-58-23](https://github.com/NalianAchei2020/rail_blog_app/assets/99715640/808a7fae-a58a-4c46-9841-c0c395e5e1d3)